### PR TITLE
Hanbok

### PR DESCRIPTION
**Pull request by toasterpm87**
# Description

Description.

---

# TODO

- [ ] Task
- [x] Completed Task

---

<details><summary><h1>Media</h1></summary>
<p>

</p>
</details>

---

# Changelog

:cl:
- add: Added fun!
- tweak: Tweaked fun!
- fix: Fixed fun!
- remove: Removed fun!

### CODE REVIEW

**Comment by toasterpm87** [comment]
Hanbok

**Review by DEATHB4DEFEAT — APPROVED** [review]
lgtm, need to see what psp thinks about the Caldera folder

**Comment by toasterpm87** [comment]
> lgtm, need to see what psp thinks about the Caldera namespace

What's igtm?

**Comment by DEATHB4DEFEAT** [comment]
> > lgtm, need to see what psp thinks about the Caldera namespace
> 
> What's igtm?

Looks
Good
To
Me

File: Resources/Textures/Caldera/Clothing/Hanbok.rsi/icon.png
Comment (Pspritechologist) [inline Resources/Textures/Caldera/Clothing/Hanbok.rsi/icon.png:1]: This should be the outfit folded up, not just the front-facing sprite.

File: Resources/Prototypes/Caldera/Clothing/Outfits [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- type: entity
+  parent: ClothingUniformSkirtBase
+  id: ClothingHanbok
+  description: A traditional Korean dress
```
Comment (Pspritechologist) [inline Resources/Prototypes/Caldera/Clothing/Outfits:4]: This item seems to not have a name?

File: Resources/Textures/Caldera/Clothing/Hanbok.rsi/Equipped-INNERCLOTHING.png
Comment (Pspritechologist) [inline Resources/Textures/Caldera/Clothing/Hanbok.rsi/Equipped-INNERCLOTHING.png:1]: The RSI directory name should not have any capitals in it.

File: Resources/Textures/Caldera/Clothing/Hanbok.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Plyush/katayusha",
```
Comment (Pspritechologist) [inline Resources/Textures/Caldera/Clothing/Hanbok.rsi/meta.json:4]: This should make clear that you can be contacted on Discord under this name.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]
Caldera namespace is probably fine? Assuming it's actually in the repo, it makes updating it from the source easier. 
Bit odd to have just the one entity, but

**Comment by toasterpm87** [comment]
> Caldera namespace is probably fine? Assuming it's actually in the repo, it makes updating it from the source easier. 
> Bit odd to have just the one entity, but

I will move more things into there.

File: Resources/Textures/Caldera/Clothing/Hanbok.rsi/icon.png
Comment (DEATHB4DEFEAT) [reply Resources/Textures/Caldera/Clothing/Hanbok.rsi/icon.png:1]: It should but I don't think any others do that

**Comment by toasterpm87** [comment]
i resolved all thingys

**Comment by DEATHB4DEFEAT** [comment]
> i resolved all thingys

You don't seem to have pushed changes

File: Resources/Textures/Caldera/Clothing/Hanbok.rsi/icon.png
Comment (Pspritechologist) [reply Resources/Textures/Caldera/Clothing/Hanbok.rsi/icon.png:1]: Lots do

**Comment by Pspritechologist** [comment]
You've marked everything as resolved, but have not actually updated anything in this pr.

**Comment by toasterpm87** [comment]
> You've marked everything as resolved, but have not actually updated anything in this pr.

Lmao I think I forgot to pr

**Comment by toasterpm87** [comment]
im gonna rewrite this

**Pull request closed without merging**